pkg/server/selectprovider/templates.go

```diff
@@ -188,23 +193,21 @@ body { background-color: var(--pf-global--BackgroundColor--dark-100); }
         <main class="pf-c-login__main">
           <div class="pf-c-login__main-body">
             {{ if eq (len .Providers) 1}}
-              <a class="pf-c-button pf-m-primary pf-m-block" href="{{ (index .Providers 0).URL }}">Log In</a>
+              <a class="pf-c-button pf-m-primary pf-m-block" href="{{ (index .Providers 0).URL }}">{{ .Locale.LogIn }}</a>
             {{ else }}
-              <h1 class="pf-c-title pf-m-3xl">Log in with&hellip;</h1>
+              <h1 class="pf-c-title pf-m-3xl">{{ .Locale.LogInWith }}</h1>
               <ul>
                 {{ range $provider := .Providers }}
                   <li class="idp">
-                    <a href="{{$provider.URL}}" class="pf-c-button pf-m-secondary pf-m-block" title="Log in with {{$provider.Name}}">{{$provider.Name}}</a>
+                    <a href="{{$provider.URL}}" class="pf-c-button pf-m-secondary pf-m-block" title="{{ .Locale.LogInWith }} {{$provider.Name}}">{{$provider.Name}}</a>
```

Comment:
The translators generally need full sentences. I think this may cause issues because it's broken up.

Reply:
@rebeccaalpert so we will need to have atranslations for the `Log In With` string  together with all the product variations ?

Reply:
I think that's safer, yeah. Like Sam mentioned above sometimes the ordering of the sentence can change depending on the language.

Reply:
The platform name is now included with the key string for the translators 
`"Welcome to {{platformTitle}}": "Welcome to {{platformTitle}}",`
https://github.com/openshift/console/pull/8587#discussion_r609747067

Reply:
Thank you!

Reply:
@sg00dwin @rebeccaalpert I'm not sure we can make that work in this context. The OAuth server won't know which platform name to use. Branding is done by replacing the entire login page template.

We might have to use string concatenation for now since we don't have the i18next library here.